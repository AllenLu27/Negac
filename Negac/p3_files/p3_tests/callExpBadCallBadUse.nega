int fn(){
	int a;
	a = fn(2);
}
