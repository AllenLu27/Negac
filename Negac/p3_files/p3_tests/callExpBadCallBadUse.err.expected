FATAL [3,6]: Function call with wrong number of args
Semantic analysis failed
